### ISSUE TITLE
Remove the TCP Access Code Features

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#14892]

**Comment by TracyLinder** [comment]
We're going to pull this out of the TCP-CMU upgrade epic as it ended up not being in the project MVP. We'll move this to backlog sprint sized work.

**Comment by TracyLinder** [comment]
@ChrispinP do you still need to do anything on the backend to remove this feature, or can we close this issue?

**Comment by ChrispinP** [comment]
@TracyLinder yes, I would still need to do some cleanup to remove this feature. We can put on deck when ready.

**Comment by ChrispinP** [comment]
This cleanup has been performed for both TCP & CSWZ objects.

Complete, closing
